created team platform, app east and west example goals (#3210)
PostHog/posthog.com: contents/handbook/people/team-structure/platform.md

## People

[See team structure page](/handbook/people/team-structure/team-structure)

## Overall goal for app and platform

### Objective

PostHog can handle 1bn events/day

### Motivation

The highest value users are likely to have high volumes. No-one in our enterprise pipeline has more volume than this.

### Owner

Karl

### Key results

> - P95 <15 seconds for Suprdaily

Reviewer: ```suggestion
- P95 query latency is < 15 seconds for our biggest customer (by event count)
```

> - Successful load test with Billion event queries

Reviewer: ```suggestion
- Our synthetic test suite successfully works against a PostHog installation with billion events
```

## Sub team - Ingestion

### Objective

PostHog ingests events at scale, fast and accurately.

### Motivation

Fast and reliable ingestion and accurate data are critical for businesses to trust and continue to use our products.

### Objective owner

James G

### Key results

> - P90 Ingestion <30s by June

Reviewer: ```suggestion
- P90 ingestion latency is < 30s by June
```

Reviewer: Nit: end of June or June 1st?

> - Minimal data integrity issues reported caused by compromise from enabling scale by June

Reviewer: Same as below but can we get a metric to keep track of this? Tags in Papercups?

## Sub team - Infrastructure

### Objective

There's a single, maintainable way to deploy PostHog at scale.

### Motivation

Self-hosting is a key part of our unique value proposition for large and enterprise companies to retain control of their data. We need to offer this in a reliable and scalable way.

### Objective owner

Guido

### Key results

- PostHog Cloud is using our standard helm chart and stable by June

> - Zero frustrated customers due to deployment and maintenance issues from Scale and Enterprise customers by June

Reviewer: I think this could be nice but do we have an actual metric tracking this? I'm not sure if 0 is a valid number considering I don't think we know what's the current baseline (if the current number is 2 is ok, if the current number is 20 probably less).

Reviewer: I don't have a good alternative except for a generic "there isn't a significant customer churn due to deployment and maintenance issues" but that's true also now I think.

Author: I think that's reasonable - maintaining this as true is important

> - Zero frustrated customers due to deployment and maintenance issues from Scale and Enterprise customers by June

Author: ```suggestion
- No significant customer churn due to deployment and maintenance issues from Scale and Enterprise customers by June
```

> - We're capable of spinning up and maintaining multiple production grade environments (eg staging/EU/private tenant) regardless of business demand

Reviewer: I would keep out the "maintaining part" as doing that will require a staffed oncall rotation and we don't have enough people in the team to be able to do that.
